guides/WRITING-DOCUMENTATION.md

```diff
@@ -54,156 +68,114 @@ The description should describe what pattern does or should be used for in plain
+List commonly confused use cases where an alternative component would be a better solution.
 
-- **Usability**
-  - **When to use**
-    - List typical use cases for this component
-  - **When to consider alternatives**
-    - List commonly confused use cases where an alternative component would be a better solution
-  - **General guidance**
-    - List any design or content guidelines related to how this component should be used
-- **Accessibility**
-  - List any accessibility considerations like required keyboard functionality, ARIA attributes, etc...
-- **Theming (optional)**
-  - List any Sass variables a developer could override to customize the appearance of the component
-- **Related patterns (optional)**
-  - List any related components or utility classes that connect with the component, whether as a parent, sibling, or child within the pattern's taxonomy.
-- **Learn more**
-  - List any links that served as inspiration, references, or research related to this component.
+### Usage
 
-## Sample page
+List any design or content guidelines related to how this component should be used.
 
-The example below is a new documentation page with a URL path of `/components/buttons`.
+### Accessibility
 
-<details>
-<summary>View sample page</summary>
+List any accessibility considerations like required keyboard functionality, ARIA attributes, etc...
 
-```scss
-/*
-Buttons
+### Theming (optional)
 
-Use buttons to signal actions.
+List any Sass variables a developer could override to customize the appearance of the component.
 
-@react-example Button.example.jsx
+### Related patterns
 
-@react-props Button.jsx
+List any related components or utility classes that connect with the component, whether as a parent, sibling, or child within the pattern's taxonomy.
 
-Markup: button.example.html
+### Learn more 
 
-.ds-c-button--primary - The primary call-to-action
+List any links that served as inspiration, references, or research related to this component.
 
-Style guide: components.buttons
-*/
-
-.ds-c-button {
-  ...
-}
-
-.ds-c-button--primary {
-  ...
-}
-
-/*
----
-## Guidance
-
-content here
+Style guide: components.component-name.guidance
 */
 ```
 
 </details>
 
-#### Additional references
-
-- [Build a Style Guide Straight from Sass](https://css-tricks.com/build-style-guide-straight-sass)
-
 ## Documenting React components
 
-Using [`react-docgen`](https://github.com/reactjs/react-docgen), we extract documentation from comments in React component files. The documentation can include an overall description and a description for each of the component's `propTypes`.
-**Note:** Documentation can be written using [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+Using [`react-docgen`](https://github.com/reactjs/react-docgen), we extract a description for each of the component's `propTypes` from comments in React component files.
 
 ### Description
```

Comment:
Can we remove this heading? Description doesnt really mean anything in the context of this section